functions/go/starlark/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.15-alpine3.12
+ENV CGO_ENABLED=0
+WORKDIR /go/src/
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+COPY main.go main.go
+COPY config.go config.go
+RUN go build -o /usr/local/bin/star ./
+
+#############################################
+
+FROM alpine:3.12
```

Comment:
where' the interpreter for the python code?

Reply:
Though Starlark is a dialect of Python, I don't think we should support Python in this function.
To clarify, this function and `kpt fn run --enable-star` both use the lib from kyaml to support Starlark. The kyaml library uses [`go.starlark.net/starlark`](https://github.com/google/starlark-go) as the Starlark interpreter.